plugin.audio.rco/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.rco" name="Het Koninklijk Concertgebouworkest" version="1.0.1" provider-name="bosel">
```

Comment:
Please bump the version to 1.0.2 due to https://github.com/xbmc/repo-plugins/pull/973